packages/gatsby/src/schema/infer/is-file.ts

```diff
@@ -0,0 +1,102 @@
+import path from "path"
+import { slash } from "gatsby-core-utils"
```

Comment:
I don't get any types coming in from this module, even though it's been converted.

Reply:
It's possible the .d.ts files aren't published alongside the package -- worth looking into!

Reply:
Ah! I had to run the `prepare` task in that package to get the types.